home-tasks/src/test/java/com/epam/tc/hw2/ex1/ExerciseSoftAssertsTest.java

```diff
@@ -0,0 +1,107 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.constants.ConstantsForUserAndTests;
+import com.epam.tc.hw2.utils.LogIn;
+import com.epam.tc.hw2.utils.SeleniumAbstractCore;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class ExerciseSoftAssertsTest extends SeleniumAbstractCore {
+
+    @Test
+    public void checkAndSearchElementsOnPageWithSoftAsserts() {
+        SoftAssert softAssert = new SoftAssert();
```

Comment:
To fluent read variable name `softAssert` can be replaced by `softly`, to have something like `softly.assertThat(...)`
```suggestion
        SoftAssertions softly = new SoftAssertions();
```